tests/cms/utils/test_disable_hix_post_save_signal.py

```diff
@@ -0,0 +1,47 @@
+from integreat_cms.cms.models import Language, Page, PageTranslation, Region
+
+from ...utils import disable_hix_post_save_signal
+
+
+def test_disable_hix_post_save_signal(db: None) -> None:
+    with disable_hix_post_save_signal():
+        demo_page_translation = create_demo_page_translation()
+
+    assert demo_page_translation.hix_score == 16
+
+
+def test_rule_out_false_positive(db: None) -> None:
+    demo_page_translation = create_demo_page_translation()
+
+    assert demo_page_translation.hix_score is None
```

Comment:
This tests that in hix signals the hix score will be set to none because `hix_enabled` of the region is False, right?
If so, then maybe it makes sense to add `hix_enabled=False` when creating the region, just to be explicit.